esmvalcore/cmor/_fixes/native6/era5_land.py

```diff
@@ -0,0 +1,127 @@
+"""Fixes for ERA5-Land."""
+import datetime
+import logging
+
+import iris
+import numpy as np
+
+from esmvalcore.cmor._fixes.native6.era5 import (get_frequency,
+                                                 fix_hourly_time_coordinate,
+                                                 fix_accumulated_units,
+                                                 multiply_with_density,
+                                                 remove_time_coordinate,
+                                                 divide_by_gravity)
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+
+class Orog(Fix):
```

Comment:
This PR adds support for precipitation only right? Can we leave orography out, or should we change the scope of the PR? Note that orography is probably the most nasty one, so I'm in favour of dropping it.

Reply:
We can leave it out at the moment, I was just worried it could be needed for every variable?

Reply:
No I think it's just another variable